src/govuk-prototype-kit/govuk-prototype-kit.config.mjs

```diff
@@ -20,7 +20,7 @@ export default async () => {
         .filter(filterPath([`${componentName}/macro.njk`]))
 
       return {
-        importFrom: slash(relative(configPaths.src, join(configPaths.components, macroPath))),
+        importFrom: slash(relative(paths.src, join(paths.src, 'govuk/components', macroPath))),
```

Comment:
**question** It's not your change, but isn't that equivalent to `join('govuk/components', macroPath)`?

Reply:
@romaricpascal Spot on! Hidden away by `configPaths.components` until now

Will fix before merging, thanks again